textproc/gpsbabel/Portfile

```
@@ -26,63 +21,59 @@ long_description    GPSBabel converts GPS waypoints, tracks, and routes \
 platforms           darwin
 homepage            https://www.gpsbabel.org
 
-depends_lib-append  port:expat \
-                    port:libusb-compat
+checksums           rmd160  dd41329723a41749a6ef929e0766fba708509116 \
+                    sha256  a5ef3b1744488d86d438291df2d65135cb04c435680fe7f64f5a94a3814ebea4 \
+                    size 16720596
```

Can the size be intended to match the other checksum types, please?
```suggestion
                    size    16720596
```

---

done and pushed